YgoProPatcher/Client.cs

```diff
@@ -436,7 +435,7 @@ private void DownloadUsingCDB(List<string> listOfDownloadedCDBS, string destinat
                     {
                         DataClass db = new DataClass(cdb);
                         DataTable dt = db.SelectQuery("SELECT id FROM datas");
-                        Status.Invoke(new Action(() => Status.Text = "Updating Pics and Scripts using " + Path.GetFileName(cdb)));
+                        Status.Invoke(new Action(() => Status.Text = "Updating pictures and scripts using " + Path.GetFileName(cdb)));
```

Comment:
Take in considerstion the amount of space it will take on the patcher’s window. Text may go out of the box if it’s too long

Reply:
I did take it in consideration. You can't know without trying. <-<